### PR TITLE
FOGL-1426 - Fix south plugin description

### PR DESCRIPTION
**Pull request by amarendra-dianomic**

### CODE REVIEW

File: python/foglamp/services/south/server.py [hunk 1]
```diff
@@ -113,7 +114,8 @@ def __init__(self):
             # Configuration handling - updates the configuration using information specific to the plugin
             config_payload = json.dumps({
                 "key": category,
-                "description": '{} Device'.format(self._name),
+                "description": '{} South Service'.format(self._name) if not ('plugin' in default_config and
```
Comment (praveen-garg) [inline python/foglamp/services/south/server.py]: Service > Plugin

File: python/foglamp/services/south/server.py [hunk 2]
```diff
@@ -79,7 +79,8 @@ def __init__(self):
             # Configuration handling - initial configuration
             category = self._name
             config = self._DEFAULT_CONFIG
-            config_descr = '{} Device'.format(self._name)
+            config_descr = '{} South Service'.format(self._name) if not ('plugin' in config and
```
Comment (praveen-garg) [inline python/foglamp/services/south/server.py]: Service > Plugin

**Comment by amarendra-dianomic** [comment]
@praveen-garg changes made per discussion. Pl check.

**Comment by ashish-jabble** [comment]
@amarendra-dianomic Please fix unit tests too as per changes for below files
```
plugins/south/cc2650async/test_cc2650async.py                                                                                 
plugins/south/cc2650poll/test_cc2650poll.py
```

**Pull request merged**